src/routes/get-started/Contribute.tsx

```diff
@@ -180,14 +180,14 @@ export default function ContributePage() {
             Use the following commands to run your playground environments:
             <br />
             <b>Angular</b>
-            <CodeSnippet lang="css" allowCopy={true} code={`npm run dev:angular`}/>
+            <CodeSnippet lang="css" allowCopy={true} code={`npm run serve:angular`}/>
```

Comment:
They will also need to run the `npm run dev:watch` for both React and Angular

Reply:
The line (172) `npm run build` can be removed.

Reply:
@chrisolsen These have been taken care of